Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -1621,12 +1621,6 @@ struct WebCore::DataListSuggestionInformation {
 };
 #endif
 
-struct WebCore::SecurityOriginData {
-    [Validator='!protocol->isHashTableDeletedValue()'] String protocol;
-    String host;
-    std::optional<uint16_t> port;
-};
-
```

Comment:
See comments above.

Reply:
I am having trouble findings earlier review feedback in the Github UI. If I remember correctly, the issue is that this required a SecurityOriginData constructor which took [protocol, host, port, opaqueIdentifier] and such constructor didn't make much sense since you should never be passing a [protocol, host, port] when you have an opaqueIdentifier.

I wanted 2 constructors, one that takes a [protocol, host, port] and another that takes an opaqueIdentifier.